extension/jaegerremotesampling/README.md

```diff
@@ -10,21 +10,26 @@ Note that the port `14250` will clash with the Jaeger Receiver. When both are us
 
 Although this extension is derived from Jaeger, it can be used by any clients who can consume this standard, such as the [OpenTelemetry Java SDK](https://github.com/open-telemetry/opentelemetry-java/tree/v1.9.1/sdk-extensions/jaeger-remote-sampler).
 
+At this moment, the `reload_interval` option is only effective for the `file` source. In the future, this property will be used to control a local cache for a `remote` source.
+
+The `file` source can be used to load files from the local file system or from remote HTTP/S sources. The `remote` source must be used with a gRPC server that provides a Jaeger remote sampling service.
+
 ## Configuration
 
 ```yaml
 extensions:
   jaegerremotesampling:
-    grpc:
```

Comment:
This component hasn't been published yet. The examples in the readme are there only for us to define and check how it would look like for an end-user when the component is finished.